serialisation-library/src/main/java/gaffer/serialisation/simple/TypeValueSerialiser.java

```diff
@@ -88,6 +90,11 @@ public TypeValue deserialise(final byte[] bytes) throws SerialisationException {
     }
 
     @Override
+    public TypeValue deserialiseEmptyBytes() throws SerialisationException {
+        return new TypeValue();
```

Comment:
I think this should be null. In the serialise method, if the type value is blank an exception is thrown - so 'new TypeValue()' would not be able to be reserialised.